gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/ddl/GfxdLRUDUnit.java

```diff
@@ -329,6 +331,10 @@ public String description() {
     waitForCriterion(waitCond, 120000, 500, true);
   }
 
+  public static void setDummytestBytes() {
+    HeapMemoryMonitor.setTestBytesUsedForThresholdSet(100);
+  }
+
```

Comment:
This is a static so needs to be cleared out in tearDown else will interfere with other tests (disable with -1).